std_msgs/String task
std_msgs/String req
---
zeabus_vision/vision_cash_in_your_chip data
